crates/sui-graphql-rpc/src/config.rs

```diff
@@ -484,14 +491,16 @@ impl Default for Limits {
     fn default() -> Self {
         // Picked so that TS SDK shim layer queries all pass limit.
         // TODO: calculate proper cost limits
+        let max_page_size = 50;
         Self {
             max_query_depth: 20,
             max_query_nodes: 300,
             max_output_nodes: 100_000,
             max_query_payload_size: 5_000,
             max_db_query_cost: 20_000,
             default_page_size: 20,
-            max_page_size: 50,
+            max_page_size,
+            max_multi_get_objects_keys: max_page_size, // keep the same as max_page_size
```

Comment:
- Let's not make this the same as `max_page_size`, the reason we factored it out into its own config is that it can/should be different -- we can start by setting this to 500 or something.
- Let's move it next to `max_transaction_ids` below, because that's what it is semantically closest to.